docs/modules/graphql.md

```diff
@@ -374,3 +374,31 @@ export default () =>
       }),
     })
 ```
+
+## Utilities
+
+### addFields(_query_, _path_, _fields_)
+
+As the name of utility says, you can use `addFields()` utility to additionally include more fields into any GraphQL **query** request.
+
+```javascript
+addFields(query, 'products.items', {
```

Comment:
Add fields is used mainly in extension, so please use an example with extension.